.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Please make sure to **read the following list** before creating a new issue:
+
+* Do you have a question on how to use vis.js? Please don't file an issue but ask your question e.g on [stackoverflow](https://stackoverflow.com/questions/tagged/vis.js) by using the tag 'vis.js' We will close use-specific questions!
+* Have you already used the [github search](https://github.com/almende/vis/issues), read the [documentation](http://visjs.org/) and looked at the [examples](https://github.com/almende/vis/tree/develop/examples)?
+* Make sure to mention with vis-component (network, timeline, graph2D, graph3d) you are referring to.
+* Make sure to use the [latest version of vis.js](https://cdnjs.com/libraries/vis) for bug reports.
```

Comment:
Puh, depends...latest version is ok here i think. maybe we have to ask some people to try the current branch.